server/api/dbConfig/seeds.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS posts;
```

Comment:
When you have drop table in here it means that every time the server resets it will delete all the inputted information in the database. When commenting it out it doesn't do that.

Reply:
Might be different though when actually deployed rather than through docker, not sure about that

Reply:
yeah when server restarts its a clean slate for everybody

Reply:
Only reason I mentioned it is because in the requirements it says `The user should be able to access their post using that show path even after a server restart`